src/plugin.js

```diff
@@ -12,6 +12,4 @@ if (globalVue) {
   globalVue.use({ install });
 }
 
-export default {
-  install,
-};
+export { VueEllipseProgress, install };
```

Comment:
@pngcsgta 

many thanks for taking your time! Unfortunately, your solution introduces breaking change as you replace the default export with an export. I would suggest you just to add below your changes the following line:
```js
export default install;
```
Then everything should be fine. It would be nice if you also could double check everything. It's also important that the component still working over CDN.

Reply:
Thanks, I pushed a new commit with that